setup.py

```diff
@@ -55,7 +55,7 @@
             "pytest",
             "pytest-cov",
             "pytest-rerunfailures",
-            "pytest-xdist",
+            "pytest-xdist[psutil]",
```

Comment:
What is the psutil addition for?

Reply:
This is the extension for using the logical processors.